### DISCUSSION TITLE
Any one running LossLess-Cut on Raspberry Pi or Orange Pi

### DISCUSSION THREAD

**Discussion by defencedog** [mifi/lossless-cut#1231] [General]
Title says it all plz mention which OS u are running it. 

Orange Pi 5 LTS is heavily powerful even to do video editing http://www.orangepi.org/html/hardWare/computerAndMicrocontrollers/details/Orange-Pi-5.html

**Comment by mifi** [comment]
Hi. That's a good idea. I will try to add build support for armv7l.
